cfgov/v1/views/reports.py

```diff
@@ -151,3 +179,53 @@ def get_queryset(self):
             .order_by("-created_at")
             .prefetch_related("tags")
         )
+
+
+class EnforcementActionsReportView(ReportView):
+    header_icon = "doc-full"
+    title = "Enforcement actions report"
+
+    list_export = [
+        "title",
+        "content.all",
+        "categories.all",
+        "court",
+        "docket_numbers.all",
```

Comment:
With my suggested @property above in the diff, this can be:

```suggestion
        "docket_number_strings",
```